src/components/TitleTransferPanel/TitleView/TitleView.tsx

```diff
@@ -7,13 +7,13 @@ import { useAddressBook } from "../../../common/hooks/useAddressBook";
 
 interface TitleViewProps {
   role: string;
-  address: string;
+  address?: string;
```

Comment:
I think it's the point I raised above => what's the expectation on the address, always there or not ?

I think this component should always expect an address and it shouldn't be optional

Reply:
Hmmm okay I've changed it back, its not clear which should it be though.